mobile/CmpE451_App/src/component/PostDetail.js

```diff
@@ -0,0 +1,215 @@
+              data={textFieldNames}
+              renderItem={({item}) => (
+                <View>
+                  <Text style={styles.fieldName}>{item.name}</Text>
+                  <Text style={styles.content}>{item.value}</Text>
+                  <Text></Text>
+                </View>
+              )}
+            />
+          </View>
+          <View>
+          <FlatList
+            showsHorizontalScrollIndicator={false}
+            data={numberFieldNames}
+            renderItem={({item}) => (
+              <View>
+                <Text style={styles.fieldName}>{item.name}</Text>
+                <Text style={styles.content}>{item.value}</Text>
+                <Text></Text>
+              </View>
+            )}
+          />
+          </View>
+          <View>
+          <FlatList
+            showsHorizontalScrollIndicator={false}
+            data={dateFieldNames}
+            renderItem={({item}) => (
+              <View>
+                <Text style={styles.fieldName}>{item.name}</Text>
+                <Text style={styles.content}>{moment(item.value).format('MMMM Do YYYY, h:mm:ss a')}</Text>
+                <Text></Text>
+              </View>
+            )}
+          />
+          </View>
+          <View>
+          <FlatList
+            showsHorizontalScrollIndicator={false}
+            data={linkFieldNames}
+            renderItem={({item}) => (
+              <View>
+                <Text style={styles.fieldName}>{item.name}</Text>
+                <TouchableOpacity onPress={() => Linking.openURL(item.value)}>
+                  <Text style={{color: COLORS.buttonColor}}>{item.value}</Text>
+                </TouchableOpacity>
+                <Text></Text>
+              </View>
+            )}
+          />
+          </View>
+
+          <View>
+          <FlatList
+            showsHorizontalScrollIndicator={false}
+            data={locationFieldNames}
+            renderItem={({item,index}) => (
+              <View>
+                <View>
+                  <Text style={styles.fieldName}>{locationFieldNames[index].name}</Text>
+                  <Text style={styles.content}>{locationFieldNames[index].value.description}</Text>
+                </View>
+                <View>
+                  <MapView
+                    style={styles.map}
+                    initialRegion={{
+                        latitude: locationFieldNames[index]['value']['geo']['latitude'],
+                        longitude: locationFieldNames[index]['value']['geo']['longitude'],
+                        latitudeDelta: 0.004757,
+                        longitudeDelta: 0.006866,
+                      }}
+                    >
+                  </MapView>
+                  <Text></Text>
+                  <Text></Text>
+                  <Text></Text>
+                  <Text></Text>
+                  <Text></Text>
+                  <Text></Text>
+                  <Text></Text>
```

Comment:
I could not understand these fields

Reply:
Map didn't show up in the screen and I found it as a solution.